src/sentry/integrations/utils/code_mapping.py

```diff
@@ -96,25 +102,22 @@ def frame_type(self) -> str:
         return type
 
     def _packaged_logic(self, frame_file_path: str) -> None:
-        if frame_file_path.find("/") > -1:
```

Comment:
This change is a left shift + dropping the else case. This screenshot may help see it:
<img width="1097" alt="image" src="https://user-images.githubusercontent.com/44410/206470787-0aa7e5ea-6b3a-450d-8051-1ec99d11efb7.png">